kernel/timeout.c

```diff
@@ -83,8 +83,18 @@ static s32_t next_timeout(void)
 	return ret;
 }
 
-void z_add_timeout(struct _timeout *to, _timeout_func_t fn, s32_t ticks)
+void z_add_timeout(struct _timeout *to, _timeout_func_t fn, k_timeout_t t)
```

Comment:
later on local variable `struct _timeout *t;` is created. Those names should not conflict.

Reply:
Good point.  I'm wondering why that collision isn't a warning....  Will fix.